### ISSUE TITLE
CMake can't find openssl 1.1 on windows

### ISSUE THREAD

**Issue by woogerboy21** [Cockatrice/Cockatrice#2424]
So far it looks like when trying to compile the application on windows using the current instructions the newest version of the OpenSSL package can not be found by CMAKE as well as the current location for the mysql DB library files are becoming stale.  The locations could use some updating.

**Comment by ctrlaltca** [comment]
The openssl guys decided to rename the libraries in openssl 1.1 because, you know, that's how you keep developers busy and happy to work with your project.

CMake addd support for this change a few months ago: https://gitlab.kitware.com/cmake/cmake/commit/ed1758f8eb58a4e52acf0f3885f82403814f5ffd

Qt is ucrrently using openssl 1.0.2, and should support openssl 1.1 only starting from Qt 5.10: https://bugreports.qt.io/browse/QTBUG-52905

We are currently using a custom cmake script to find openssl libraries on windows, but we'll probably only need to add the new library names: https://github.com/Cockatrice/Cockatrice/blob/master/cmake/FindWin32SslRuntime.cmake#L52

tl;dr: stick with openssl 1.0.2 by now.

**Comment by ctrlaltca** [comment]
Added a note on the wiki: https://github.com/Cockatrice/Cockatrice/wiki/Compiling-Cockatrice-(Windows)
